code/modules/reagents/chemistry/reagents/medicine.dm

```diff
@@ -593,6 +593,8 @@
 	color = "#5BCBE1"
 	addiction_chance = 1
 	addiction_threshold = 10
+	allowed_overdose_process = TRUE
```

Comment:
Any particular reason it should process the regular effects while OD'd?

Reply:
I feel like it couldn't hurt for it to still be good at what it's supposed to do (purging histamine), plus the sleepiness effects are good to include imho

Reply:
I don't see any hurt in it processing while overdosing 👍